.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
You can add `.idea` to your [global gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore) -- we don't want to end up with a gitignore for over 9000 IDEs and tools here :)
```suggestion
```